omod/src/main/java/org/openmrs/module/adminui/page/controller/myaccount/ChangeDefaultsPageController.java

```diff
@@ -81,12 +140,149 @@ public void setDefaultLocale(String defaultLocale) {
             this.defaultLocale = defaultLocale;
         }
 
+        public void setProficientLocales(String proficientLocales) {
+        	setProficientLocalesToSet(proficientLocales);
+        }
+
         public String getProficientLocales() {
-            return proficientLocales;
+        	return getProficientLocalesFromSet();
         }
 
-        public void setProficientLocales(String proficientLocales) {
-            this.proficientLocales = proficientLocales;
+        public void setDisplayProficientLocales(Set<Locale> displayLocales, String proficientLocales) {
+            setDisplayLocales(displayLocales);
+            setProficientLocales(proficientLocales);
+        }
+
+        public void setDisplayProficientLocalesList(Set<Locale> displayLocales, String[] proficientLocalesList) {
+            setDisplayLocales(displayLocales);
+            setProficientLocalesList(proficientLocalesList);
+        }
+
+        public Set<LocaleOption> getProficientLocalesSet() {
+            return proficientLocalesSet;
+        }
+
+        public void setProficientLocalesSet(Set<LocaleOption> proficientLocalesSet) {
+            this.proficientLocalesSet = proficientLocalesSet;
+            if (displayLocales != null)
```

Comment:
Our code convention is that the open brace should be on the same line as the if clause, though i would assume the formatter would fix it when building the project.